### DISCUSSION TITLE
How to use RTSP(UDP-multicast) on multiple network cards?

### DISCUSSION THREAD

**Discussion by ssBJT** [bluenviron/mediamtx#1963] [Questions]
I have two network cards.
PC1, use command.
```
ffmpeg -f gdigrab -framerate 30 -i desktop -c:v h264_nvenc -f rtsp rtsp://192.168.0.100:8554/live
```
![image](https://github.com/bluenviron/mediamtx/assets/96523008/cd9b9b69-3240-4c6b-a82f-38e971c74909)

If both are enabled. 
In PC2, IP: 192.168.0.101, use command
```
ffplay.exe -rtsp_transport udp_multicast "rtsp://192.168.0.100:8554/live"
```
can not play.

If disable one.
![image](https://github.com/bluenviron/mediamtx/assets/96523008/14214132-ff9b-41e5-a4a7-fc47e9a836c9)
PC2 can play!

Another network cards cannot be disabled.

How to choose a network card to use UDP multicast? 
Or other solution?

Thanks.

**Comment by aler9** [comment]
Hello, i gathered some documentation and it seems that on Windows multicast packets are routed to a single interface at a time, as confirmed by your post. Therefore, the server has to be edited in order to allow choosing the multicast interface or to send packets to all interfaces in parallel. Probably the best option is the second one, since it allows any client to read streams with multicast, regardless of their interface. I've created an issue here: #2029
